assets/events.ts

```diff
@@ -173,10 +173,36 @@ export function tabs() {
     for (let i = 0; i < wrappers.length; i++) {
       const labels = wrappers[i].querySelectorAll(".tab-label");
       const tabs = wrappers[i].querySelectorAll(".tab");
+      const defaultTab = wrappers[i].querySelector(".tab.tab-default");
 
       if (tabs.length > 0) {
-        // Set the first tab in a group to display
-        (tabs[0] as HTMLElement).style.display = "block";
+        // if a tab has been specified as default, set that
+        // as active as opposed to defaulting to the first tab
+        if (defaultTab) {
+          // changes an id (i.e tab-js-esm-6f3904f86f90c21d) into just the type (tab-js-esm)
+          // by removing the last element from the split array and then re-joining it
+          const parts = defaultTab.id.split("-");
+          const tabId = parts.slice(0, parts.length - 1).join("-");
+
+          let defaultTabLabel;
+          let found = false;
+          labels.forEach((label) => {
+            const labelId = label.getAttribute("data-link");
+            if (!found) {
+              if (labelId === tabId) { 
+                defaultTabLabel = label; 
+                found = true;
+               }
+            }
```

Comment:
no need for an inner if
```suggestion
            if (!found && labelId === tabId) {
              defaultTabLabel = label; 
              found = true;
            }
```